lib/bundle/brew_dumper.rb

```diff
@@ -17,12 +17,10 @@ def self.formulae
 
     def self.dump
       formulae.map do |f|
-        if f[:args].empty?
-          "brew '#{f[:full_name]}'"
-        else
-          args = f[:args].map { |arg| "'#{arg}'" }.sort.join(", ")
-          "brew '#{f[:full_name]}', args: [#{args}]"
-        end
+        brewline = "brew '#{f[:full_name]}'"
+        args = f[:args].map { |arg| "'#{arg}'" }.sort.join(", ")
+        brewline += ", args: [#{args}]" unless f[:args].empty?
+        brewline += BrewServices.started?(f[:full_name]) ? ", service_restart: true" : ''
```

Comment:
Thu Aug 11 08:52:56 BST 2016

```
irb(main):009:0> s = 'string'
=> "string"
irb(main):010:0> s += ' is da word' if true
=> "string is da word"
irb(main):011:0> s = 'string'
=> "string"
irb(main):012:0> s += ' is da word' if false
=> nil
irb(main):013:0>
irb(main):017:0> s = 'string'
=> "string"
irb(main):018:0> s += false ?  ' is true' : ' is false'
=> "string is false"
irb(main):019:0> s = 'string'
=> "string"
irb(main):020:0> s += true ?  ' is true' : ' is false'
=> "string is true"
```

Proves why the blank lines in ```Brewfile```,  but don't understand the ```why```.
(Something to do with **everything is an expression** in Ruby.)

### A-Ha moment?? ###
It's ```nil``` because the test fails, so the implicit ```else``` is used, which is ```nil```.